SeaPublicWebsite.UnitTests/PropertyDataServiceTests.cs

```diff
@@ -84,25 +84,4 @@ public async Task UpdatePropertyDataWithRecommendations_WhenCalledOnReferenceTha
         // Assert
         Assert.AreEqual(testTime, returnedPropertyData.RecommendationsFirstRetrievedAt);
     }
-    
-    [Test]
-    public async Task UpdatePropertyDataWithRecommendations_WhenCalledOnReferenceThatHasNoTimestamp_ReturnsPropertyDataWithTimestampForNow()
```

Comment:
Should we have a test for the new behaviour?

Reply:
I didn't because there wasn't a file for it already, but you're right that there ought to be